torch/csrc/jit/tensorexpr/llvm_jit.cpp

```diff
@@ -75,6 +76,68 @@ class TORCH_API PytorchLLVMJITImpl {
     cantFail(LLJ->defineAbsolute(
         *Mangle("remainderf"),
         {llvm::pointerToJITTargetAddress(&remainderf), {}}));
+
+    // FP32 Sleef functions -- SSE
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_acosf4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_acosf4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_asinf4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_asinf4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_atanf4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_atanf4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_atan2f4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_atan2f4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_cosf4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_cosf4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_sinf4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_sinf4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_tanf4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_tanf4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_coshf4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_coshf4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_sinhf4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_sinhf4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_tanhf4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_tanhf4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_erff4_u10"),
+        {llvm::pointerToJITTargetAddress(&Sleef_erff4_u10), {}}));
+    cantFail(LLJ->defineAbsolute(
+        *Mangle("Sleef_erfcf4_u15"),
```

Comment:
Thinking about this more: why don't we bind the Sleef symbols to a name that omits the ulp suffix?  That way we don't need the override for `erfc`